usercase: add basic cli usecases (#4801)
ElektraInitiative/libelektra: doc/usecases/cli/UC_get_config.md

# Use Case: cli get configuration

## Summary

- **Scope:** CLI use

> - **Level:** Developer Goal, UI

Reviewer: Please use one of the following values, as defined in the template https://github.com/ElektraInitiative/libelektra/blob/master/doc/usecases/template.md:

- Cloud
- Kite
- Sea
- Underwater
- Clam

See this PR for details: https://github.com/ElektraInitiative/libelektra/pull/4762/files#r1045125432

This applies also to the two other use cases.

- **Actors:** CLI user
- **Brief:** User reads data for a given keyname

## Scenarios

- **Precondition:** -
- **Main success scenario:**

>   - Backend is asked for the data for the given keyname

Reviewer: ```suggestion
  - The CLI user queries data from the backend by providing a keyname.
```
The scenario should mention the actors (CLI user in this case) and their actions.

>   - the data is outputted

Reviewer: ```suggestion
  - The Backend returns the data and the CLI tool displays it.
```

- **Alternative scenario:** -
- **Error scenario:**
  - The keyname does not exist
  - The keyname is not valid

>   - The keyname contains a bookmark that does not exist

Reviewer: Please also describe what happens in this cases, e.g. that an error message is displayed.

- **Postcondition:**

>   - Data for keyname or error-message is outputted

Reviewer 2: Error scenarios are usually not part of the postcondition (as other use cases written now). If you want to be more precise, split up postcondition to minimal guarantee and success guarantee. (See Writing effective Use cases).

```suggestion
  - Value of key is printed
```
